BetterStats/BetterStats.cs

```diff
@@ -665,7 +674,12 @@ public static void AddPlanetFactoryData(PlanetFactory planetFactory)
                 if (assembler.id != i || assembler.recipeId == 0) continue;
 
                 var frequency = 60f / (float)((double)assembler.timeSpend / 600000.0);
-                var speed = (float)(0.0001 * (double)assembler.speed);
+                var speed = (float)(0.0001 * Math.Max(assembler.speedOverride, assembler.speed));
+                // forceAccMode is true when Production Speedup is selected
+                if (!assembler.forceAccMode)
+                {
+                    frequency += 60f / (float)( assembler.extraTimeSpend / 600000.0);
```

Comment:
This part is to account for the extra products that are created in "Extra Products" mode

Reply:
Thx, it can work on my computer.